### ISSUE TITLE
upgrade to tldraw 2.3

### ISSUE THREAD

**Issue by farvardin** [YunoHost-Apps/tldraw_ynh#115]
Hello, tldraw is now at the 2.3.0 version, will you upgrade to this one? At the moment it's not possible to export projects from the official tldraw website and import them on the yunohost version, probably because the yunohost one (1.24.5) is too old.

**Comment by Salamandar** [comment]
tl;dr tldraw is not a webapp anymore (it's a library), and it's not as easy as before to self host the example app.
See #116

**Comment by farvardin** [comment]
@Salamandar I've tried to install your v2 branch on my server (sudo yunohost app install https://github.com/YunoHost-Apps/tldraw_ynh/tree/v2 --debug) but all I got was a 500 error.

**Comment by Salamandar** [comment]
Yes, the code base of tldraw completely changed and is basically impossible to install now.
